src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -7,6 +7,7 @@ import {
   customElement,
 } from "lit-element";
 import "@polymer/paper-icon-button/paper-icon-button";
+import "@thomasloven/round-slider";
```

Comment:
Should this be dynamically imported?

Reply:
no. It's tiny.

Reply:
The old one was huge as it required all of jQuery to be added too.

Reply:
yeah, a massive improvement, I'm sure